Examples/OrchestratorTenantMigration.ps1

```diff
@@ -0,0 +1,692 @@
+﻿<###########################################################
+
+README FIRTS
```

Comment:
Please use proper PowerShell comment tags. See https://github.com/UiPath/orchestrator-powershell/blob/master/Examples/Sync-UiPathADUsers.ps1#L2 for an example.
Read https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7 for how the PowerShell comment based help works.